src/content/getting-started/quickstart/openshift/gcp-lb/_index.md

```diff
@@ -1,7 +1,7 @@
 ---
 date: 2020-02-21T13:36:18+01:00
 title: "On GCP (LoadBalancer mode)"
-weight: 20
+weight: 35
```

Comment:
Reserving `30` for `Azure`. So the layout would be `On AWS` -> `On AWS with Globalnet` -> `On Azure` -> `On GCP` -> `Hybrid ....` arranged alphabetically.